manifests/client.pp

```diff
@@ -241,6 +246,14 @@
       ensure => 'running'
   }
 
+  if ($include_clientlaunch_d) {
+    file_line { 'include_clientlaunch_d':
+      path => "${xymon_config_dir}/clientlaunch.cfg",
+      line => "directory ${actual_clientlaunch_config}",
+    }
+    File[$actual_clientlaunch_config] -> File_line[include_clientlaunch_d] -> Service[$service_name]
```

Comment:
Shouldn't this be ~> Service?

Reply:
Hm, yes, makes sense. Will change it.

Reply:
...and fixed another error i saw here with missing quotes on the File_line resource...